src/yb/server/default-path-handlers.cc

```diff
@@ -118,12 +119,12 @@ struct Tags {
 // Writes the last FLAGS_web_log_bytes of the INFO logfile to a webpage
 // Note to get best performance, set GLOG_logbuflevel=-1 to prevent log buffering
 static void LogsHandler(const Webserver::WebRequest& req, Webserver::WebResponse* resp) {
-  std::stringstream *output = &resp->output;
+  std::stringstream* output = &resp->output;
```

Comment:
prefer not to have unrelated changes in this commit. this could be a separate commit if needed.

Reply:
Same as above